scripts/format.sh

```diff
@@ -7,32 +7,23 @@ set -o errexit -o pipefail -o nounset
 #  - 1 there are files to be formatted
 #  - 0 everything looks fine
 
-
-readonly CLANG_FORMAT_VERSION=11.0.0
+# see https://pypi.org/project/clang-format/#history
+readonly CLANG_FORMAT_VERSION=11.0.1
 
 if [[ $(uname -i) == 'aarch64' ]]; then
   echo 'Formatting is disabled on arm for the time being'
   exit
 fi
 source scripts/bash_utils.sh
-setup_mason
-
-./mason/mason install clang-format $CLANG_FORMAT_VERSION
-./mason/mason link clang-format $CLANG_FORMAT_VERSION
-readonly CLANG_FORMAT=$(pwd)/mason_packages/.link/bin/clang-format
-
-echo "Using clang-format $CLANG_FORMAT_VERSION from ${CLANG_FORMAT}"
-
-find src valhalla test -type f -name '*.h' -o -name '*.cc' \
-  | xargs -I{} -P ${NPROC} ${CLANG_FORMAT} -i -style=file {}
 
 # Python setup
 py=$(setup_python)
-if [[ $(python3 -m pip list | grep -c "black\|flake8") -ne 2 ]]; then
-  if [[ $(python3 -c 'import sys; print(int(sys.base_prefix != sys.prefix or hasattr(sys, "real_prefix")))') -eq 1 ]]; then
-    ${py} -m pip install black==24.10.0 flake8==7.1.1
+CLANG_FORMAT_CMD="${py} -c \"from clang_format import clang_format; clang_format()\""
```

Comment:
it took a while to understand how to make sure we'll use the right `clang-format`, i.e. with full path. other things didn't work. on circleci `clang-format` is installed to `/usr/local/bin` but all python paths suggest everything is installed in the `/usr` namespace. anyways, the package has a weird way to run the executable directly at least..